File: Python/2670.py
n1 = int(input())
n2 = int(input())
n3 = int(input())

if n1+n3 <= 2*n2:
    print(2*n1 + 2*n3)
elif n1 > n2 and n1 > n3:
    print(2*n2 + 4*n3)
elif n3 > n1 and n3 > n2:
    print(4*n1 + 2*n2)

File: JavaScript/2670.js
// Problema: 2670 - Máquina de Café  | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s] | Tempo: 0.250s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var [n1, n2, n3] = lines.splice(0,3).map(Number);

var r;
if (n1 + n3 <= 2 * n2) {r = (n1+n3) * 2; }
else if (Math.max(n1, n2, n3) == n1){ r = 2*n2 + 4*n3; }
else if (Math.max(n1, n2, n3) == n3){ r = 4*n1 + 2*n2; }

console.log(r);
